src/VecSim/CMakeLists.txt

```diff
@@ -5,11 +5,35 @@ set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
 set(CMAKE_CXX_STANDARD 20)
 
 project(VecsimLib)
+
 file(GLOB_RECURSE headers ./**.h)
 set(HEADER_LIST "${headers}")
 
 include_directories(../)
 
+set(SVS_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++20 -march=native -mtune=native")
+set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3")
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g")
+
+# option(SVS_EXPERIMENTAL_LEANVEC "Enable experimental LeanVec in SVS" YES)
+# if(SVS_EXPERIMENTAL_LEANVEC)
+#     add_definitions(-DSVS_EXPERIMENTAL_LEANVEC)
+# endif()
+
+set(SVS_REPOSITORY "https://github.com/intel/ScalableVectorSearch.git" CACHE STRING "SVS repository")
+
+include(FetchContent)
```

Comment:
Is there a particular reason to use `FetchContent` rather than using a submodule?

Reply:
We just followed existing VecSim approach: there is no `.gitmodules` but `googletest`, `google_benchmark`, `pybind11` linked using `FetchContent`

Reply:
As discussed, since svs is a substantial submodule that is not only required for testing, we will want to have it as a git submodule to have a more convenient mechanism for tracking and changing the versions.

Reply:
I've switched SVS to submodule, but got PR builds fail because submodule update is not run in CI builds.

Reply:
should be fixed now